_posts/2019-03-07-strimzi-0.11.0-release.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title:  "Strimzi 0.11.0 and 0.11.1 released!"
+date: 2018-03-07
+author: paolo_patierno
+---
+
+We are delighted to announce the new Strimzi 0.11.0 and 0.11.1 releases with some new features!
```

Comment:
Why announce 0.11.0 at all? I would just announce 0.11.1.

Reply:
@scholzj suggested that most of the new features were in 0.11.0